src/core.jl

```diff
@@ -36,7 +38,7 @@ macro required(keywords...)
     for keyword in keywords
         push!(blocks, quote
             err_msg = string($(string(keyword)), " is required")
-            $(esc(keyword)) === nothing && error(err_msg)
+            $(esc(keyword)) === nothing && @error(err_msg)
```

Comment:
FWIW, `error` still works and throws an error and is probably what you want here. `@error` is a logging macro.